src/physics/utilities/equation_solver.cpp

```diff
@@ -163,16 +163,18 @@ mfem::Operator& EquationSolver::SuperLUNonlinearOperatorWrapper::GetGradient(con
   return *superlu_grad_mat_;
 }
 
-void EquationSolver::defineInputFileSchema(std::shared_ptr<axom::inlet::SchemaCreator> schema_creator)
+void EquationSolver::defineInputFileSchema(axom::inlet::Table& table)
 {
-  auto nonlinear_table = schema_creator->addTable("nonlinear", "Newton Equation Solver Parameters")->required(true);
+  auto nonlinear_table = table.addTable("nonlinear", "Newton Equation Solver Parameters");
+  nonlinear_table->required(false);
```

Comment:
Did you mean to flip this to required=false?

Reply:
Yeah, in general an `EquationSolver` doesn't needed nonlinearparameters - it's just the solid solver that needs it.